buildchain/static-container-registry/static-container-registry.py

```diff
@@ -88,8 +88,9 @@ def find_images(root):
             yield (name, tag)
 
 
-def create_config(root, server_root, name_prefix):
-    yield CONSTANTS
+def create_config(root, server_root, name_prefix, with_constants=True):
```

Comment:
We  probably want to patch upstream as well, no?

Reply:
Upstream ? I did not know it was a downloaded script.
But btw I patched this only for our use case, if we want to keep it the same as upstream I can also do some "filtering" directly in the doit task

Reply:
[Upstream](https://github.com/NicolasT/static-container-registry/) is from @NicolasT so you can see with him if he want the change or not.

Reply:
I'll take care of 'upstreaming' and then re-syncing our subtree at a later point in time, never mind for now.